src/Chartjs/Resources/assets/dist/controller.js

```diff
@@ -63,7 +65,7 @@ var _default = /*#__PURE__*/function (_Controller) {
         options: payload.options
       });
 
-      var chart = new _chart.Chart(this.element.getContext('2d'), payload);
+      var chart = new _auto["default"](this.element.getContext('2d'), payload);
```

Comment:
From tests:
```js
var chart = new _auto["default"](this.element.getContext('2d'), payload);
```
ReferenceError: ResizeObserver is not defined.

But from one application woks well